posts/2020-10-23-beta-20.0.0.12.adoc

```diff
@@ -0,0 +1,452 @@
+
+* `ConfigSource.getPropertyNames` is no longer a default method. The implementation of a `ConfigSource` must implement this method.
+* Previous versions of the specification would not evaluate Property Expressions. As such, previous working configuration may behave differently (if the used configuration contains values with Property Expressions syntax). Property Expressions can be disabled by setting the property `mp.config.property.expressions.enabled` with the value of `false`.
+* Empty values or other special characters are no longer valid Config property values. This means that `Config.getValue(String propertyName, Class<T> propertyType)` will never return `null`; a `NoSuchElementException` will be thrown if either a Config property value is not defined, is defined as an empty string, or its converter returns `null`. In the previous release, the empty value was returned as an empty value. For more information, please refer to this link:https://download.eclipse.org/microprofile/microprofile-config-2.0-RC1/microprofile-config-spec.html#_config_value_conversion_rules[section].
+
+To enable the new beta features in your app, add them to your `server.xml`:
+
+[source, xml]
+----
+<featureManager>
+   <feature>mpConfig-2.0</feature>
+</featureManager>
+----
+
+For more information:
+
+link:https://download.eclipse.org/microprofile/microprofile-config-2.0-RC1/microprofile-config-spec.html#release_notes_20[Release Notes for MicroProfile Config 2.0]
+
+link:https://github.com/eclipse/microprofile-config/milestone/8?closed=1[MicroProfile Config 2.0 Milestone]
+
+[#jwt]
+=== Support for MicroProfile JWT 1.2
+
+The MicroProfile JSON Web Token 1.2 specification allows using a JWT token for authenticating and authorizing requests to a service. The specification simplifies the configuration for managing the validation of the JWT by introducing new MicroProfile Config properties. Enhanced signature algorithm support is added in this Open Liberty beta.
+
+Version 1.2 of the MicroProfile JSON Web Token specification adds the following MicroProfile Config properties to control different aspects of the JWT validation.
+
+==== mp.jwt.token.header
+
+The `mp.jwt.token.header` property allows you to control the HTTP request header which is expected to contain a JWT token. You can specify either Authorization (default) or Cookie values.
+
+==== mp.jwt.token.cookie
+
+The `mp.jwt.token.cookie` property allows you to specify the name of the cookie which is expected to contain a JWT token. The default value is `Bearer` if not specified.
+
+==== mp.jwt.verify.audiences
+
+The `mp.jwt.verify.audiences` property allows you to create list of allowable audience (`aud`) values. At least one of these must be found in the claim. Previously, this had to be configured in the `server.xml` file. Now, you can configure the audiences in the MicroProfile Config property as follows:
+
+[source, java]
```

Comment:
think this should just be "text" not "java" as these are entries in the config properties file.